examples/camera_server/camera_server.cpp

```diff
@@ -23,7 +11,7 @@ int main(int argc, char** argv)
 
     // 14030 is the default camera port for PX4 SITL
     // FIXME: the camera should probably initiate the connection and not just listen
-    auto result = mavsdk.add_any_connection("udp://:14030");
+    auto result = mavsdk.add_any_connection("udp://127.0.0.1:14030");
```

Comment:
I think that's correct. Thanks! 

Can you please remove the comment above it though?